src/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,118 @@
+import userEvent from '@testing-library/user-event';
+import { describe, it, expect, vi } from 'vitest';
+
+import { render, screen } from '../../test/test-utils';
+
+import { Button } from './Button';
+
+describe('Button', () => {
+  it('renderiza con props por defecto', () => {
+    render(<Button data-testid="btn">Hola</Button>);
+    const btn = screen.getByTestId('btn');
```

Comment:
Tests use many getByTestId queries (example here). Project testing guidelines prefer semantic queries (getByRole, getByLabelText) over test ids when possible. Using getByTestId everywhere reduces test robustness and misses accessibility coverage. getByTestId usages appear repeatedly (lines 11, 25, 36, 49, 64, 76, 88, 100, 114-116).

**Sugerencia:**
```typescript
Prefer semantic selectors. Example: `screen.getByRole('button', { name: /Click/i })` for button content, or `getByLabelText('sample-input')` for textareas with aria-label. Keep data-testid only for elements that can't be selected semantically.
```